apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -194,7 +201,7 @@ ManageAssets.propTypes = {
 };
 
 export function ManageModels(props) {
-  return <PopUpMenu.Item {...props}>{'Manage Models'}</PopUpMenu.Item>;
+  return <PopUpMenu.Item {...props}>{'Manage A.I. Models'}</PopUpMenu.Item>;
```

Comment:
At least in Oceans, we used "A.I." as the bot's name, and AI for the concept.  Do we want to do same here?

Reply:
should we move this string to an i18n file so we don't have to remember to go back and make things translatable?